.github/workflows/bump-elastic-stack-snapshot.yml

```diff
@@ -31,33 +31,14 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - name: Install Updatecli in the runner
-        uses: updatecli/updatecli-action@339fdb3f3bf66d14fe5830248b4e893990415cd0 # v2.80.0
-
-      - name: Run Updatecli in Apply mode
-        run: updatecli --experimental apply --config .github/workflows/updatecli.d/bump-elastic-stack-snapshot.yml --values .github/workflows/updatecli.d/values.d/scm.yml
+      - uses: elastic/oblt-actions/updatecli/run-and-notify@v1
```

Comment:
We maintain `elastic/oblt-actions` so we can avoid the burden of bumping versions